src/seraphis_impl/serialization_demo_types.h

```diff
@@ -481,10 +484,385 @@ struct ser_JamtisPaymentProposalSelfSendV1 final
     END_SERIALIZE()
 };
 
+
+// EnoteStore serialization
+
+// LegacyEnote types
+struct ser_LegacyEnoteV1 final
```

Comment:
I just want to point out that after PR https://github.com/monero-project/monero/pull/9069, you can actually write free functions for serialization, which means that you can serialize directly from that function and don't have to create new types.

Reply:
Yes, you can do that. And for non-blob types, you can write a `do_serialize()` function. I might make changes to the serialization header to make this an easier process.

Reply:
Actually it might be better to leave as-is for now so it matches with the rest of the file's serialization. I will probably refactor all these classes out later